modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/adapters/AdapterRegistry.scala

```diff
@@ -34,29 +34,46 @@ import com.snowplowanalytics.snowplow.enrich.common.utils.HttpClient
  * The AdapterRegistry lets us convert a CollectorPayload into one or more RawEvents, using a given
  * adapter.
  */
-class AdapterRegistry(remoteAdapters: Map[(String, String), RemoteAdapter] = Map.empty) {
-
+class AdapterRegistry(
+  remoteAdapters: Map[(String, String), RemoteAdapter] = Map.empty,
+  callrailConfig: CallrailConfig,
+  cloudfrontAccessLogConfig: CloudfrontAccessLogConfig,
+  googleAnalyticsConfig: GoogleAnalyticsConfig,
+  hubspotConfig: HubspotConfig,
+  mailchimpConfig: MailchimpConfig,
+  mailgunConfig: MailgunConfig,
+  mandrillConfig: MandrillConfig,
+  marketoConfig: MarketoConfig,
+  olarkConfig: OlarkConfig,
+  pagerdutyConfig: PagerdutyConfig,
+  pingdomConfig: PingdomConfig,
+  sendgridConfig: SendgridConfig,
+  statusGatorConfig: StatusGatorConfig,
+  unbounceConfig: UnbounceConfig,
+  urbanAirshipConfig: UrbanAirshipConfig,
+  veroConfig: VeroConfig
+) {
```

Comment:
```suggestion
class AdapterRegistry(
  remoteAdapters: Map[(String, String), RemoteAdapter] = Map.empty,
  schemas: AdaptersSchemas
) {
```

Reply:
Done! Also good idea on renaming `AdaptersConfig` to `AdaptersSchemas`. Have also changed this in the configuration files to use the key `adaptersSchemas` instead of `adapters` since that only configures the schemas for adapters.